### PR TITLE
disable network in docker images

### PR DESCRIPTION
**Pull request by GuyAfik**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
add the label [tests_require_network] to the .pack-ignore of packs that their scripts/integrations require docker network

## Must have
- [x] Tests
- [ ] Documentation

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -9,7 +9,7 @@ license = "MIT"
 python = "^3.8,<3.11"
 
 [tool.poetry.group.dev.dependencies]
-demisto-sdk = "1.10.1"
+demisto-sdk = {git = "https://github.com/demisto/demisto-sdk.git", rev = "disable_network_lint"}
```
Comment (JudahSchwartz) [inline pyproject.toml:12]: why not do a release?

Comment (GuyAfik) [reply pyproject.toml:12]: not ready for a review

**Review by JudahSchwartz — APPROVED** [review]

**Comment by GuyAfik** [comment]
lint -a looks good with the changes from the SDK: https://code.pan.run/xsoar/content/-/pipelines/5066934

**Comment by GuyAfik** [comment]
@dorschw @JudahSchwartz please re-review as its been stale for quite some time.

**Review by JudahSchwartz — APPROVED** [review]
merge this first

**Pull request merged**